brainstormforce/astra
# fix: Removed double :not selectors & solved the Mega Menu issues with Old Header.

### Description
 - Reduced the :not selector CSS.
 - Mega Menu breaks with Old Header.

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Switch to the old Header & check Mega Menu functionality with all the Widths.
 - Switch to New Header & check the same as 1st point.
 - In Toggle for Desktop the mega Menu should be converted to a normal menu.

### Checklist:
